test/fork/migration-v2.js

```diff
@@ -440,17 +424,41 @@ describe('v2 migration', function () {
       this.master.address,
     );
     await stakingPool.deployed();
-
-    this.stakingPoolFactory = stakingPoolFactory;
     this.stakingPool = stakingPool;
   });
 
-  it('deploy & upgrade contracts: CR, TC, MCR, MR, CO, PS, P1, GW, CoverMigrator', async function () {
-    const coverProxyAddress = await this.master.contractAddresses(toUtf8Bytes('CO'));
+  // TODO: What is this for?!
```

Comment:
Tests that the arbitrary test data CLAIM_PAYABLE_ADDRESS is wiped out for MEMBER_ADDRESS as part of the storageCleanup function call for MemberRoles, post upgrade.

This is just to setup test data to allow for verifying that getClaimPayoutAddress returns AddressZero post cleanup, when MEMBER_ADDRESS is passed as a parameter.


Suggestions for clarity: add a comment explaining this + rename MEMBER_ADDRESS to something like `MEMBER_WITH_CLEARABLE_PAYOUT_ADDRESS`